src/foam/mlang/mlang.js

```diff
@@ -1685,6 +1687,9 @@ foam.CLASS({
       return this[name].create({ arg1: arg1, arg2: arg2 });
     },
 
+    function TRUE() { return this.True.create(); },
```

Comment:
I got 'Uncaught (in promise) TypeError: Cannot read property 'create' of undefined' in this line. Is this function used in other places without a bind?